keyboards/gh60/satan/keymaps/pcoves/rules.mk

```diff
@@ -0,0 +1,22 @@
+# Build Options
```

Comment:
You don't need this whole ruleset here. You can keep it if you want, but this could be condensed down to:

```Makefile
BOOTMAGIC_ENABLE       = no  # Virtual DIP switch configuration
MOUSEKEY_ENABLE        = yes # Mouse keys
CONSOLE_ENABLE         = no  # Console for debug
SLEEP_LED_ENABLE       = no  # Breathing sleep LED during USB suspend
RGBLIGHT_ENABLE        = no  # Enable keyboard RGB underglow

COMBO_ENABLE           = yes
RAINBOW_UNICORN_ENABLE = yes
TAP_DANCE_ENABLE       = yes
UNICODE_ENABLE         = yes
```

Reply:
And done :-)